base/bitarray.jl

```diff
@@ -600,7 +600,6 @@ function insert!(B::BitVector, i::Integer, item)
 end
 
 function _deleteat!(B::BitVector, i::Integer)
-
     k, j = get_chunks_id(i)
```

Comment:
Please don't delete blank lines in cases like this where the author has clearly taken some care to use whitespace to delineate functional blocks.